src/xunit.performance.execution/BenchmarkEventSource.cs

```diff
@@ -191,14 +201,24 @@ public unsafe void BenchmarkIterationStop(string RunId, string BenchmarkName, in
                 }
             }
 
-            if (_csvWriter != null)
+            if (s_csvWriter != null)
                 WriteCSV(BenchmarkName, iteration: Iteration);
         }
 
-        public static readonly BenchmarkEventSource Log = new BenchmarkEventSource();
+        private BenchmarkEventSource() : base(true)
```

Comment:
Yes, it does cause EventSource to throw if there is an error on the underlying window code. I can remove it as I instrumented the API to find out why CoreCLR repo stopped reporting events.